### ISSUE TITLE
Missing test: `fix-whitespace` + `make test` + `stack.yaml`

### ISSUE THREAD

**Issue by asr** [agda/agda#4864]
@rwe, in #4853 you fixed the issue with `fix-whitespace` + `make test` + `stack.yaml`. Thanks!

Running the following commands

```bash
$ cp stack-XYZ.yaml stack.yaml
$ stack build fix-whitespace
```

I'm getting some compilation errors do not detect by TravisI. ¿Could you add a test for your fix, please?

**Comment by rwe** [comment]
@asr Can you please describe the errors? What version is XYZ? And are they fixed by https://github.com/agda/fix-whitespace/pull/8 ? (Which needs review and merge, I don't have permissions on that repo).

**Comment by asr** [comment]
I was using GHC 8.10.2. I was wrong, the error shouldn't be detected by Travis, but a GitHub workflow. At the moment we are not using GHC 8.10.2 on GitHub workflows due to https://github.com/actions/setup-haskell/pull/32. I merged  [agda/fix-whitespace#](https://github.com/agda/fix-whitespace/pull/8), thanks!
